applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, Model, settings):
             for node in cond.GetNodes():
                 node.Set(KratosMultiphysics.SOLID)
 
-    def ExecuteInitialize(self):
+    def ExecuteInitializeSolutionStep(self):
```

Comment:
Could you add a flag to decide whether to define the wake in each step or only once at the beginning? As you know, for some compressible and transonic cases we need to do load stepping to achieve convergence but it is not necessary to compute the wake in each step but rather only once at the beginning. In 2D defining the wake doesn't take much time but in 3D it is more expensive.

Reply:
True, I actually like much more the option to activate/deactivate this behaviour. Thanks for the hint. Also, we should add the multi-step option in the apply_far_field_process for the transonic cases.